Lib/test/test_fileinput.py

```diff
@@ -686,7 +702,6 @@ def do_test_call_input(self):
         self.assertIs(files, result.files, "files")
         self.assertIs(inplace, result.inplace, "inplace")
         self.assertIs(backup, result.backup, "backup")
-        self.assertIs(bufsize, result.bufsize, "bufsize")
```

Comment:
Please restore this assertion.

Reply:
This is testing the Mock, the MockFileInput have a bufsize attribute, the rel FileInput does not. The goal of this test is to explicitly test that `fileinput.input` is forwrding ìnut(bufsize=...)` to `FileInput(bufsize...)` which this patch does remove in order to print a warnign with the correct stack. I can change to assertIsNot though.